### DISCUSSION TITLE
Get Error java.sql.SQLException: Row size too large. The maximum row size for the used table type, not counting BLOBs, is 8126. This includes storage overhead, check the manual.

### DISCUSSION THREAD

**Discussion by husmannm** [Keyfactor/ejbca-ce#194] [Q&A]
I'm trying to update from EJBCA 6.15.2.1 Community (r31915) to ejbca-ce-7-10-0-2.
Now I'm getting the error
`java.sql.SQLException: Row size too large. The maximum row size for the used table type, not counting BLOBs, is 8126. This includes storage overhead, check the manual. You have to change some columns to TEXT or BLOBs`
on startup of wildfly.

No idea what's the reason for this error.
Is this a known error?
Which tables I have to change?

Server: Debian Bullseye 11.5
DB: MariaDB 10.5.18
WildFly Full 10.1.0.Final
EJBCA: CE 7.10.0.2

[server.zip](https://github.com/Keyfactor/ejbca-ce/files/10238982/server.zip)

Any hint on this for me?
Thanks in advance.

**Comment by primetomas** [comment]
It should be a database configuration. It's normally not 8192 if you would install a "standard" MariaDB today I think. This article have some hints, i.e. rowtype. Make sure you are using InnoDB, etc.
https://matomo.org/faq/troubleshooting/how-do-i-fix-the-error-1118-row-size-too-large-the-maximum-row-size-for-the-used-table-type-not-counting-blobs-is-8126/

There's tone of settings you can do on the database.
We have some documentation here, but not on the row format though.
https://doc.primekey.com/ejbca/ejbca-installation/creating-the-database
